src/Bicep.Core.Samples/InvalidVariables_LF/main.diagnostics.bicep

```diff
@@ -120,3 +120,13 @@ var objWithInterp = {
   'ab${nonExistentIdentifier}cd': true
 //@[7:28) [BCP057 (Error)] The name 'nonExistentIdentifier' does not exist in the current context. |nonExistentIdentifier|
 }
+
+// invalid fully qualified function access
+var mySum = az.add(1,2)
+//@[15:18) [BCP057 (Error)] The name 'add' does not exist in the current context. |add|
+var myConcat = sys.concat('a', az.concat('b', 'c'))
+//@[34:40) [BCP057 (Error)] The name 'concat' does not exist in the current context. |concat|
```

Comment:
Could you add a test to validate the error message returned if a user (for example) creates a var named `resourceGroup` and later tries to access it as a function without the namespace qualifier? (you may already have one - sorry if I missed it). I feel like this could be quite confusing for users, so want to make sure the error message is as clear as possible.

```
var resourceGroup = 'myRg'
...
var somethingElse = resourceGroup().location
```

Reply:
Done, also added new negative and positive tests